recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
     - patches/0001-remove-pyproject.toml-requires-in-setup.py.patch
 
 build:
-  number: 0
+  number: 1
```

Comment:
This does not need a version number bump.  Please reset to zero

Reply:
I know that the build doesn't change, but I wasn't sure what happens e.g. if build jobs time out after merging to master. Removing